### ISSUE TITLE
Feature Pitch : Archers can't hang trapblock

### ISSUE THREAD

**Issue by Orsovi** [transhumandesign/kag-base#116]
Actually, archers can hang with grapplin on trapblock whitch it a bit odd because themself can go through it.

So I propose to let act trapblock on grapplin as on archers.

**Comment by terracraft321** [comment]
That's true but it is really useful for something like the parkour modification. Thus this might need a complete rework of the mapping.
